docs/source/getting_started.rst

```diff
@@ -0,0 +1,185 @@
+.. _getting_started:
+
+===============
+Getting started
+===============
+
+.. _prerequisites:
+
+**************
+Pre-requisites
+**************
+To make full use of pyQuil, you'll want to have both the Quantum Virtual Machine (QVM) and the Quil Compiler (quilc) installed. If you
+don't have those installed yet, refer to `Rigetti's guide on installing the Quil SDK locally <https://docs.rigetti.com/qcs/getting-started/installing-locally>`_.
```

Comment:
This page previously contained a lot of documentation on installing and running `quilc` and `qvm` on various platforms. I've chosen to remove that and defer to the Rigetti docs instead.